# DTO
### Publication: Drug target ontology to classify and integrate drug discovery data (https://jbiomedsem.biomedcentral.com/articles/10.1186/s13326-017-0161-x)

In the context of the Illuminating the Druggable Genome ([IDG](https://commonfund.nih.gov/idg)) program we have been developing the Drug Target Ontology (DTO). DTO integrates and harmonizes knowledge of the most important druggable protein families: kinases, GPCRs, ion channels and nuclear hormone receptors. DTO content was curated from several resources and the literature, and includes detailed hierarchical classifications of proteins and genes, tissue localization, disease association, drug target development level, protein domain information, ligands, substrates, and other types of relevant information. DTO content sources were chosen by domain experts based on relevance, coverage and completeness of the information available through them. Most resources had been peer reviewed (references are included in the respective sections), published and were therefore considered reliable. 

DTO is aimed towards the drug discovery and clinical communities and was built to align with other ontologies including BioAssay Ontology (BAO) [1, 2, 3] and GPCR Ontology [4]. By providing a semantic framework of diverse information related to druggable proteins, DTO facilitates the otherwise challenging integration and formal linking of heterogeneous and diverse data important for drug discovery. DTO is particularly relevant for big data, systems-level models of diseases and drug action as well as precision medicine. The long-term goal of DTO is to provide such an integrative framework and to populate the ontology with this information as a community resource. DTO has already been implemented in end-user software tools to facilitate the browsing [3] and navigation of drug target data [5].


### Reference:
1. Abeyruwan S, Vempati UD, Kucuk-McGinty H, Visser U, Koleti A, Mir A, Sakurai K, Chung C, Bittker JA, Clemons PA, et al. Evolving BioAssay ontology (BAO): modularization, integration and applications. J Biomed Semantics. 2014;5(Suppl 1 Proceedings of the Bio-Ontologies Spec Interest G):S5.[View Article](https://doi.org/10.1186/2041-1480-5-S1-S5) [Google Scholar](http://scholar.google.com/scholar_lookup?title=Evolving%20BioAssay%20ontology%20%28BAO%29%3A%20modularization%2C%20integration%20and%20applications&author=S.%20Abeyruwan&author=UD.%20Vempati&author=H.%20Kucuk-McGinty&author=U.%20Visser&author=A.%20Koleti&author=A.%20Mir&author=K.%20Sakurai&author=C.%20Chung&author=JA.%20Bittker&author=PA.%20Clemons&journal=J%20Biomed%20Semantics&volume=5&issue=Suppl%201%20Proceedings%20of%20the%20Bio-Ontologies%20Spec%20Interest%20G&pages=S5&publication_year=2014)
2. Vempati UD, Przydzial MJ, Chung C, Abeyruwan S, Mir A, Sakurai K, Visser U, Lemmon VP, Schurer SC. Formalization, annotation and analysis of diverse drug and probe screening assay datasets using the BioAssay ontology (BAO). PLoS One. 2012;7(11):e49198.[View Article](https://doi.org/10.1371/journal.pone.0049198) [Google Scholar](http://scholar.google.com/scholar_lookup?title=Formalization%2C%20annotation%20and%20analysis%20of%20diverse%20drug%20and%20probe%20screening%20assay%20datasets%20using%20the%20BioAssay%20ontology%20%28BAO%29&author=UD.%20Vempati&author=MJ.%20Przydzial&author=C.%20Chung&author=S.%20Abeyruwan&author=A.%20Mir&author=K.%20Sakurai&author=U.%20Visser&author=VP.%20Lemmon&author=SC.%20Schurer&journal=PLoS%20One&volume=7&issue=11&pages=e49198&publication_year=2012)
3. Nguyen DT, Mathias S, Bologa C, Brunak S, Fernandez N, Gaulton A, Hersey A, Holmes J, Jensen LJ, Karlsson A, et al. Pharos: collating protein information to shed light on the druggable genome. Nucleic Acids Res. 2017;45(D1):D995–D1002.[View Article](https://doi.org/10.1093/nar/gkw1072) [Google Scholar](http://scholar.google.com/scholar_lookup?title=Pharos%3A%20collating%20protein%20information%20to%20shed%20light%20on%20the%20druggable%20genome&author=DT.%20Nguyen&author=S.%20Mathias&author=C.%20Bologa&author=S.%20Brunak&author=N.%20Fernandez&author=A.%20Gaulton&author=A.%20Hersey&author=J.%20Holmes&author=LJ.%20Jensen&author=A.%20Karlsson&journal=Nucleic%20Acids%20Res&volume=45&issue=D1&pages=D995-D1002&publication_year=2017)
4. Przydzial MJ, Bhhatarai B, Koleti A, Vempati U, Schurer SC. GPCR ontology: development and application of a G protein-coupled receptor pharmacology knowledge framework. Bioinformatics. 2013;29(24):3211–9.[View Article](https://doi.org/10.1093/bioinformatics/btt565) [Google Scholar](http://scholar.google.com/scholar_lookup?title=GPCR%20ontology%3A%20development%20and%20application%20of%20a%20G%20protein-coupled%20receptor%20pharmacology%20knowledge%20framework&author=MJ.%20Przydzial&author=B.%20Bhhatarai&author=A.%20Koleti&author=U.%20Vempati&author=SC.%20Schurer&journal=Bioinformatics&volume=29&issue=24&pages=3211-3219&publication_year=2013)
5. Cannon DC, Yang JJ, Mathias SL, Ursu O, Mani S, Waller A, Schurer SC, Jensen LJ, Sklar LA, Bologa CG, et al. TIN-X: Target Importance and Novelty Explorer. Bioinformatics. 2017;33(16):2601–3.[Google Scholar](https://scholar.google.com/scholar?hl=en&q=Cannon%20DC%2C%20Yang%20JJ%2C%20Mathias%20SL%2C%20Ursu%20O%2C%20Mani%20S%2C%20Waller%20A%2C%20Schurer%20SC%2C%20Jensen%20LJ%2C%20Sklar%20LA%2C%20Bologa%20CG%2C%20et%20al.%20TIN-X%3A%20Target%20Importance%20and%20Novelty%20Explorer.%20Bioinformatics.%202017%3B33%2816%29%3A2601%E2%80%933.)
